packages/dm-core/src/hooks/useDocument.tsx

```diff
@@ -1,9 +1,11 @@
-import { useEffect, useState } from 'react'
 import { AxiosError } from 'axios'
+import { useEffect, useState } from 'react'
```

Comment:
I guess NotificationManager import on line 4 + ts ignore on line 3 can be removed